testing/matrix-sdk-integration-testing/README.md

```diff
@@ -20,13 +22,13 @@ The integration tests can be run with `cargo test` or `cargo nextest run`.
 
 The integration tests expect the environment variables `HOMESERVER_URL` to be the HTTP URL to
 access the synapse server and `HOMESERVER_DOMAIN` to be set to the domain configured in
-that server. If you are using the provided `docker-compose`, the default will be fine.
+that server. If you are using the provided `docker-compose.yml`, the default will be fine.
 
 ## Maintenance
 
-To drop the database of your docker-compose run:
+To drop the database of your docker compose run:
 
 ```bash
-docker-compose -f assets/docker-compose.yml stop
-rm -rf ./assets/data
+docker compose -f assets/docker-compose.yml stop
+docker volume rm assets_db assets_synapse
```

Comment:
I don't know if it's valid for docker-compose too, but seems I can run `podman compose -f assets/docker-compose.yml down --volumes` and that both shuts down the services, removes the instances and the volumes at once, which is nice, and likely what we want to do between two runs. What do you think?

Reply:
I agree

Reply:
Done